### PR TITLE
Dracula5

### PR DESCRIPTION
**Pull request by Spooky-0**
#### Checklist

* [ ] Use a separate branch in your local repo (not `master`).
* [ ] I've read and agree with the [contribution guidelines](https://www.freeboardgames.org/docs/?path=/story/documentation-contributing--page).

### CODE REVIEW

**Comment by flamecoals** [comment]
This is overall looking good! I will just do some play testing and if I don't find anything major I am going to approve :)

**Comment by flamecoals** [comment]
I played the game, and I like the simplicity of the UI with emojis :D.

The game is also working well on mobile, and given that you have good test coverage for the rules, we can publish it :).

Only a few things that could be improved and iterated in the future:
- The tap target for the emojis are small on mobile. Users might click by mistake on the wrong action.
- For players that are waiting for another player, it would be nice to show "Waiting for X..." instead of the action title.
- For the voting action, it would be nice to reiterate who is the mayor and priest on the message.
- It would be nice to have at least one UI test
- The table works now as emojis, but I believe a simple SVG would be even better.

**Comment by flamecoals** [comment]
Ok, just found a blocker: I don't see you rendering the game over page, and your tests dont include this condition either.
We can either merge the PR now but keep it in development, or fix this and publish it :).

**Review by flamecoals — CHANGES_REQUESTED** [review]

**Comment by flamecoals** [comment]
I also noticed that the `endIf` that should finish the game is mis-spelled (lower case I: `endif`)

File: web/src/games/secretDracula/board.tsx [hunk 1]
```diff
@@ -42,12 +43,11 @@ export class Board extends React.Component<IBoardProps> {
 
     return (
       <div className={css.div}>
-        TODO: [] Allow only 5-10 players [x] Fix being able to renominate (for debug i allowed it) [x] Split up main
-        render function [] Add tests suite [x] Show other players what player was investigated [x] Show players the vote
-        outcome (eg: 3yes, 4no) [] check end conditions [x] remove a lot of console.logs
         <GameLayout gameArgs={this.props.gameArgs} allowWiderScreen={true}>
```
[the remark this answers is not in the document]
Comment (flamecoals) [reply web/src/games/secretDracula/board.tsx:46]: Here is an example of rendering the game over page:
https://github.com/freeboardgames/FreeBoardGames.org/blob/8b6306b5f4fa82a9b61589cb077612c180e610bd/web/src/games/tictactoe/board.tsx#L83

Here is an example on how to check if the game is over on the test:
https://github.com/freeboardgames/FreeBoardGames.org/blob/8b6306b5f4fa82a9b61589cb077612c180e610bd/web/src/games/tictactoe/game.test.ts#L53

File: web/src/games/secretDracula/game.test.ts [hunk 2]
```diff
@@ -0,0 +1,781 @@
...
+  players.map((p, i) => p.moves.moveOKVote(i));
+  players[1].moves.moveDiscardMayor(2, 1);
+  players[0].moves.moveDiscardPriest(1, 0); // -- play blood
+
+  players[1].moves.moveExecute(0, 1);
+
+  // var {G, ctx} = players[0].getState();
+  // console.log(G)
+  // console.log(ctx)
+  // TODO: Now the game should be over, as Dracula is dead!
```
[the remark this answers is not in the document]
Comment (Spooky-0) [reply web/src/games/secretDracula/game.test.ts:90]: done.

File: web/src/games/secretDracula/board.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Spooky-0) [reply web/src/games/secretDracula/board.tsx:46]: done. Its not pretty, but done.

**Pull request closed without merging**